ChineseFilter/sections/antiadblock.txt

```diff
@@ -1,6 +1,9 @@
 !
 ! These rules are against all kinds of ad blocking reinjection/circumvention scripts
 !
+!
+! https://github.com/AdguardTeam/AdguardFilters/issues/70806
+hboav.com#%#//scriptlet('abort-current-inline-script', 'onload', 'warm_msg')
```

Comment:
Please check this `||5278.cc/js/check.js` on your end instead of the scriptlet.

Reply:
Ok, a second.

Reply:
Working, will change soon.